### ISSUE TITLE
Unsupported whereClause causes crash in server when event is raised

### ISSUE THREAD

**Issue by erossignon** [node-opcua/node-opcua#810]
**Current behavior**

( original reported in https://stackoverflow.com/questions/62313353/server-throws-error-with-prosys-and-matrikon-clients-on-events) 
Server raising an event might crash when connected by a Prosys or Matrikon Client

```
Users/brbl/Projects/GitHub/opcua-tutorial/node_modules/node-opcua-address-space/src/alarms_and_conditions/check_where_clause.ts:120
    }
     ^
Error: Only OfType operator are allowed in checkWhereClause
    at Object.checkWhereClause (/Users/brbl/Projects/GitHub/opcua-tutorial/node_modules/node-opcua-address-space/src/alarms_and_conditions/check_where_clause.ts:120:6)
    at MonitoredItem._on_opcua_event (/Users/brbl/Projects/GitHub/opcua-tutorial/node_modules/node-opcua-server/source/monitored_item.ts:833:10)
    at UAObject.emit (events.js:214:15)
    at UAObject.EventEmitter.emit (domain.js:476:20)
    at UAObject._bubble_up_event (/Users/brbl/Projects/GitHub/opcua-tutorial/node_modules/node-opcua-address-space/src/ua_object.ts:161:20)
    at UAObject.raiseEvent (/Users/brbl/Projects/GitHub/opcua-tutorial/node_modules/node-opcua-address-space/src/ua_object.ts:144:14)
    at Timeout._onTimeout (/Users/brbl/Projects/GitHub/opcua-tutorial/my_event_server.ts:95:11)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
```

**Describe the bug**
 the WhereClause emitted by the client is not well handled by node-opcua server.

**To Reproduce**
1.  start an node-opcua server that raise an event
2. . connect ProsysClient and display the event view
3.. see server crashing 

**Expected behavior**

   * Description of the other OPCUA system I am trying to connect to:
   
     - Name: Prosys or Matrikon

* **node-opcua version:**  :  >=2.50 &&  <=2.6.5

**Comment by erossignon** [comment]
the whereClause taht causes the crash is 
```
 whereClause                   /* ContentFilter       */: {
   elements                    /* ContentFilterElem[] */: [
     { /*0*/
       filterOperator          /* FilterOperator      */: FilterOperator.Not ( 7)
       filterOperands          /* ExtensionObject  [] */: [ /* length =1*/

         { /*ElementOperand*/
          index                         /* UInt32              */: 1               0x1
         };
       ]
     },
     { /*1*/
       filterOperator          /* FilterOperator      */: FilterOperator.OfType ( 14)
       filterOperands          /* ExtensionObject  [] */: [ /* length =1*/

         { /*LiteralOperand*/
          value                         /* Variant             */: Variant(Scalar<NodeId>, value: GeneralModelChangeEventType (ns=0;i=2133))
         };
       ]
     }
   ]
 }
````

 FilterOperator.Not   is currently not supported . need to be fixed

**Comment by erossignon** [comment]
fixed in node-opcua@2.5.6
